src/api/spec/factories/users.rb

```diff
@@ -2,7 +2,7 @@
   factory :user do
     email { Faker::Internet.email }
     realname { Faker::Name.name }
-    login { Faker::Internet.user_name(nil, %w(_)) }
+    sequence(:login){|n| "#{Faker::Internet.user_name(nil, %w(_))}#{n}" }
```

Comment:
If I review my last code... I always setting the login from outside the factory... so, let's use a sequence for it